content/swiftui/concepts/uikit/uikit.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
This metadata is formatted incorrectly for a concept entry. (You used the topic template, which is different.) For instructions on how to build metadata for a concept, you can read the [style guide section on metadata](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata).